include/cadmium/engine/pdevs_dynamic_runner.hpp

```diff
@@ -120,7 +122,12 @@ namespace cadmium {
                         _top_coordinator.collect_outputs(_next);
                         _top_coordinator.advance_simulation(_next);
                         _next = _top_coordinator.next();
+
```

Comment:
Maybe, we can define a `run_until(const TIME &t, bool print_log)` method, and define an additional method to keep backwards compatibility:
```c++
TIME run_until(const TIME &t) {
    return run_until(t, false);
}
```
We should do something very similar for the `run_until_passivate` method.

Reply:
If we use the two small functions at the bottom, then this function is already backwards compatible since the bool is set to false and is changed when the turn_on() function is called before run_until().